style.css

```diff
@@ -813,4 +815,603 @@ justify-content: space-between;
+        max-width: 809px;
+        width: 100%;
+        min-height: 437px;
+        padding-left: 0px;
+        padding-top: 51px;
+    }
+
+    .safety__car-frame {
+        position: relative;
+        padding-left: 0px;
+        padding-top: 100px;
+    }
+    .car-frame__image-tablet {
+        display: block;
+        width: 100%;
+        max-width: 735px;
+    }
+    .car-frame__image {
+        display: none; 
+    }
+    .car-frame__watermark-wrapper {
+        left: 90px;
+        right: 0px;
+        top: 90px;
+        max-width: 600px; 
+        }
+    .safety__raiting {
+        position: absolute;
+        top: 64px;
+        right: 86px;
+        min-height: 263px 
+    }
+
+    .about-safety__title {
+        display: inline-block;
+        max-width: 316px;
+        margin-top: 10px;
+        margin-bottom: 0px;
+        margin-left: 34px;
+    }
+
+    .about-safety__text {
+        margin-bottom: 90px;
+        margin-left: 34px;
+    }
+
+    .safety__whole-raiting {
+        display: none;
+    }
+
+    .footer__copyright {
+        display: none;
+    }
+
+}
+
+@media only screen and (min-width: 500px) and (max-width: 734px) {
+    .description {
+        margin-bottom: 487px;
+    }
+    .safety {
+        position: relative;
+        width: 100%;
+        min-height: 537px;
+        }
+    .safety__main-wrapper {
+        position: absolute;
+        left: 0px;
+        top: 560px;
+        max-width: 809px;
+        width: 100%;
+        min-height: 400px;
+        padding-left: 0px;
+        padding-top: 51px;
+    }
+    .safety__about-safety {
+        margin: 0 auto;
+        max-width: 316px;
+    }
+    .safety__car-frame {
+        position: relative;
+        padding-left: 0px;
+        padding-top: 100px;
+    }
+    .car-frame__image-tablet {
+        display: block;
+        width: 100%;
+        max-width: 735px;
+    }
+    .car-frame__image {
+        display: none; 
+    }
+    .car-frame__watermark-wrapper {
+        left: 90px;
+        right: 0px;
+        top: 90px;
+        max-width: 600px; 
+        }
+    .safety__raiting {
+        display: none;
+    }
+    .safety__whole-raiting {
+        display: none;
+    }
+
+    .footer__copyright {
+        display: none;
+    }
+}
+
+@media only screen and (max-width: 499px) {
+    .description {
+        margin-bottom: 294px;
+    }
+    .safety {
+        position: relative;
+        width: 100%;
+        min-height: 294px;
+        }
+    .safety__main-wrapper {
+        position: absolute;
+        left: 0px;
+        top: 296px;
+        max-width: 809px;
+        width: 100%;
+        min-height: 304px;
+        padding-left: 0px;
+        padding-top: 18px;
+    }
+    .safety__about-safety {
+        margin: 0 auto;
+        max-width: 316px;
+    }
+    .safety__car-frame {
+        position: relative;
+        padding-left: 0px;
+        padding-top: 10px;
+    }
+    .car-frame__image-tablet {
+        display: block;
+        width: 100%;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Angleter_tesla/pullRequest?prid=5388479)